ember-file-upload/README.md

```diff
@@ -10,16 +10,12 @@ Uploads can be managed through queues and continue in the background, even after
 
 ## Compatibility
 
-* Ember.js 3.25 or above
+* Ember.js 3.28 or above
 * TypeScript 5.0 or above
 * ember-auto-import 2.0 or above
-* Ember CLI v2.13 or above
-* Node.js v16 or above
```

Comment:
I don't have a good reference for this but since this is a v2 addon Ember CLI and Node versions are not relevant to compatibility